contracts/lib/SafeMath.sol

```diff
@@ -0,0 +1,36 @@
+pragma solidity ^0.4.18;
```

Comment:
need to use same version `0.4.24` like other files

Reply:
this was taken unmodified from zeppelin